swatch-product-configuration/src/main/resources/subscription_configs/RHEL/RHEL_for_x86_els_payg.yaml

```diff
@@ -29,9 +27,8 @@ metrics:
     awsDimension: vCPU_hours
     azureDimension: vcpu_hours
     prometheus:
-      queryKey: default
+      queryKey: rhelemeter
       queryParams:
+        productLabelRegex: .*(^|,)(204)($|,).*
```

Comment:
Can you add a comment about what the intention with this value is and where the value 204 comes from?
This is used by `product=~"#{metric.prometheus.queryParams[productLabelRegex]}"`, so I guess you want to only fetch metrics for product that contains the product ID 204, right?